mhtc-web/components/LanguageSwitcher.tsx

```
@@ -1,57 +1,128 @@
+  );
 
-  const handleLanguageChange = (langCode: string) => {
-    setCurrentLang(langCode)
-    localStorage.setItem('lang', langCode)
-    onLanguageChange(langCode)
-  }
+  // Helper function to get translated language name
+  const getTranslatedName = (code: string): string => {
+    // If translations for language names exist in common.json
+    if (t.languages && t.languages[code]) {
+      return t.languages[code];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)